subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPlugin.java

```diff
@@ -226,6 +228,26 @@ public void execute(Task task) {
         });
     }
 
+    private String getPullRequestDescription(CreatePullRequestTask task) {
+        if (StringUtil.isEmpty(task.getPullRequestDescription())) {
+            return String.format("This pull request was automatically created by Shipkit's" +
+                    " 'org.shipkit.upgrade-downstream' Gradle plugin (http://shipkit.org)." +
+                    " Please merge it so that you are using fresh version of '%s' dependency.",
+                task.getVersionUpgrade().getDependencyName());
+        } else {
+            return task.getPullRequestDescription();
+        }
+    }
+
+    private String getPullRequestTitle(CreatePullRequestTask task) {
+        if (StringUtil.isEmpty(task.getPullRequestDescription())) {
```

Comment:
The same as for the "if" in getPullRequestDescription.

Reply:
@wwilk Where should defaults be initialized then? In getter of this class in similar way I did in Upgrade Dependency Plugin but in CreatePullRequestTask? Cause I need to have access to  ${versionUpgrade.newVersion} while setting their default value

I will be thankful for hint there.

Reply:
Getter is fine here, this line:
`if (StringUtil.isEmpty(task.getPullRequestDescription()`
is redundant because it will always be empty at this point.